.gitignore

```diff
@@ -3,6 +3,7 @@
 /lib/
 
 # Windows - Visual Studio
+[Tt]emp*/
```

Comment:
I would suggest making this match the exact folder name for the build folder.

Reply:
Fixed